bin/create_maf_database.py

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env python3
+import argparse
+import logging
+
+from genie import input_to_database
+from genie import process_functions
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def main(db_mapping_id, pemfile):
+
+    syn = process_functions.synLogin(pemfile, debug=debug)
+
+    query = 'SELECT * FROM {}'.format(db_mapping_id)
+    databaseToSynIdMapping = syn.tableQuery(query)
+    databaseToSynIdMappingDf = databaseToSynIdMapping.asDataFrame()
+
+    databaseToSynIdMappingDf = \
+            input_to_database.create_and_archive_maf_database(syn, databaseToSynIdMappingDf)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--db_mapping_id', type=str,
```

Comment:
Will this just be projectid at some point since we will be adding this as an annotation?

Reply:
Will update this parameter once this is fixed: #222